enterprise/app/sidebar/sidebar.tsx

```diff
@@ -280,6 +280,7 @@ export default class SidebarComponent extends React.Component {
                   </div>
                 )}
               </div>
+              <hr />
```

Comment:
how does this look?

Reply:
Same as before -- I added an override to remove the default user agent styles:

![image](https://user-images.githubusercontent.com/2414826/145858514-c0fba0f6-9560-4d18-b6a2-08100f0ba966.png)